"do clean" complains about missing "gcc"

```
$ ./do clean
make: Entering directory 'tests/isa'
make -C ./../../ext/riscv-tests/isa XLEN=64 clean
make[1]: Entering directory 'ext/riscv-tests/isa'
Makefile:58: *** couldn't find gcc.  Stop.
make[1]: Leaving directory 'ext/riscv-tests/isa'
make: *** [Makefile:85: clean] Error 2
make: Leaving directory 'tests/isa'
make: Entering directory 'tests/isa'
make -C ./../../ext/riscv-tests/isa XLEN=32 clean
make[1]: Entering directory 'ext/riscv-tests/isa'
Makefile:58: *** couldn't find gcc.  Stop.
make[1]: Leaving directory 'ext/riscv-tests/isa'
make: *** [Makefile:85: clean] Error 2
make: Leaving directory 'tests/isa'
```

--- a/tests/isa/Makefile
+++ b/tests/isa/Makefile
@@ -85,5 +85,13 @@ riscv_tests:
 # Clean up
 
 clean:
-	$(MAKE) -C $(riscv_tests_dir)/isa XLEN=$(XLEN) clean
+	@if [ -d "$(riscv_tests_dir)/isa" ] && [ -f "$(riscv_tests_dir)/isa/Makefile" ]; then \
+		if command -v "$(RISCV_GCC)" > /dev/null 2>&1; then \
+			$(MAKE) -C $(riscv_tests_dir)/isa XLEN=$(XLEN) clean; \
+		else \
+			echo "Skipping $(riscv_tests_dir)/isa clean: $(RISCV_GCC) not found"; \
+		fi; \
+	else \
+		echo "Skipping $(riscv_tests_dir)/isa clean: submodule not initialized"; \
+	fi
 	rm -rf $(junk)

--- a/tests/isa/Makefile
+++ b/tests/isa/Makefile
@@ -85,13 +85,7 @@ riscv_tests:
 # Clean up
 
 clean:
-	@if [ -d "$(riscv_tests_dir)/isa" ] && [ -f "$(riscv_tests_dir)/isa/Makefile" ]; then \
-		if command -v "$(RISCV_GCC)" > /dev/null 2>&1; then \
-			$(MAKE) -C $(riscv_tests_dir)/isa XLEN=$(XLEN) clean; \
-		else \
-			:; \
-		fi; \
-	else \
-		:; \
+	@if [ -f "$(riscv_tests_dir)/isa/Makefile" ] && command -v "$(RISCV_GCC)" > /dev/null 2>&1; then \
+		$(MAKE) -C $(riscv_tests_dir)/isa XLEN=$(XLEN) clean; \
 	fi
 	rm -rf $(junk)

--- a/tests/isa/Makefile
+++ b/tests/isa/Makefile
@@ -89,9 +89,9 @@ clean:
 		if command -v "$(RISCV_GCC)" > /dev/null 2>&1; then \
 			$(MAKE) -C $(riscv_tests_dir)/isa XLEN=$(XLEN) clean; \
 		else \
-			echo "Skipping $(riscv_tests_dir)/isa clean: $(RISCV_GCC) not found"; \
+			:; \
 		fi; \
 	else \
-		echo "Skipping $(riscv_tests_dir)/isa clean: submodule not initialized"; \
+		:; \
 	fi
 	rm -rf $(junk)

